index.js

```diff
@@ -1,9 +1,15 @@
 var bs58check = require('bs58check')
+var bech32 = require('bech32')
 var bufferEquals = require('buffer-equals')
 var createHash = require('create-hash')
 var secp256k1 = require('secp256k1')
 var varuint = require('varuint-bitcoin')
 
+var SEGWIT_TYPES = {
+  P2WPKH: 'p2wpkh',
+  P2SH_P2WPKH: 'p2sh(p2wpkh)'
+}
```

Comment:
IMHO,  you should additionally support `P2PKH`,  as that is the default behaviour.

Reply:
I was trying not to break backwards compatibility... ie. passing nothing for segwitType will == exactly the same as before.

I could remove the compressed boolean arg and replace it with a type arg that would include segwit types AND p2pkh and p2pkh-uncompressed